templates/content-certificate.php

```diff
@@ -54,6 +54,10 @@
 				</button>
 			<?php endif; ?>
 
+			<?php if ( $is_sharing_enabled ) : ?>
+				<input readonly="readonly" id="llms_sharing_permalink" onClick="this.select();" value="<?php echo esc_url( get_permalink( get_the_ID() ) ); ?>">
```

Comment:
Ensure keyboard users can still benefit. Although it seems in practice that it's still triggered with `onClick` I like semantics of `onfocus`.

```suggestion
				<input readonly="readonly" id="llms_sharing_permalink" onfocus="this.select();" value="<?php echo esc_url( get_permalink( get_the_ID() ) ); ?>">
```